src/components/UserInfo.js

```diff
@@ -1,16 +1,24 @@
 export default class UserInfo {
-    constructor({nameElement, jobElement}) {
+    constructor({nameElement, jobElement, avatarElement}) {
         this._nameElement = document.querySelector(nameElement);
         this._jobElement = document.querySelector(jobElement);
+        this._avatarElement = document.querySelector(avatarElement);
     }
     getUserInfo() {
         return {
             name: this._nameElement.textContent,
-            description: this._jobElement.textContent
+            about: this._jobElement.textContent,
+            avatar: this._avatarElement.textContent
         }
     }
+
     setUserInfo(data) {
         this._nameElement.textContent = data.name;
-        this._jobElement.textContent = data.description;
+        this._jobElement.textContent = data.about;
+        this._avatarElement.src = data.avatar;
```

Comment:
`        this._avatarElement.src = data.avatar;
`


Please, reuse the `setAvatar` method here, to avoid code duplication


```js
    setUserInfo(data) {
        this._nameElement.textContent = data.name;
        this._jobElement.textContent = data.about;
        this.setAvatar(data.avatar)
    }


```

Reply:
My remark hasn't been corrected